src/level/robotmain.h

```diff
@@ -150,6 +150,9 @@ const int SATCOM_PROG       = 4;
 const int SATCOM_SOLUCE     = 5;
 const int SATCOM_MAX        = 6;
 
+const int CMD_HISTORY_SLOTS = 10;
+const int CMD_MAX_LENGHT = 50;
```

Comment:
1. It's LENGTH not LENGHT
2. There should be no limit to the length. We are actively working on removing all such limits and refactoring to std::string.